src/data/roadmapData.json

```diff
@@ -422,12 +422,74 @@
     {
       "CI Integration": [
         {
-          "name": "placeholder",
+          "name": "Basic CI knowledge",
           "data": [
             {
-              "description": "description 1",
-              "url": "http://google.com",
-              "urlLabel": "some urlLabel"
+              "description": "Describes Continuous Integration. The artcle contains several links to the useful resources.",
+              "url": "https://www.atlassian.com/continuous-delivery/tutorials/continuous-integration-tutorial",
+              "urlLabel": "Basic knowledge about Continuous Integration"
+            },
+            {
+              "description": "Presents types of tests and CI. Describes how to adopt Continuous Integration",
+              "url": "https://www.atlassian.com/continuous-delivery/continuous-integration/how-to-get-to-continuous-integration",
+              "urlLabel": "Continuous Integration in 5 steps"
+            },
+            {
+              "description": "How to optimize test cases for Continuous Integration",
+              "url": "https://www.browserstack.com/guide/optimize-tests-for-continuous-integration",
+              "urlLabel": "Continuous Integration and tests"
+            }
+
+          ]
+        },
+        {
+          "name": "Continuous Integration tools",
+          "data": [
+            {
+              "description": "What Are the Best Continuous Integration Tools?",
+              "url": "https://bitbar.com/blog/top-continuous-integration-tools-for-devops/",
+              "urlLabel": "Continuous Integration Tools"
+            }
+          ]
+        },
+        {
+          "name": "Continuous Integration with Jenkins",
+          "data": [
+            {
+              "description": "This article describes how to use the Jenkins continuous integration build server.",
+              "url": "https://www.vogella.com/tutorials/Jenkins/article.html",
+              "urlLabel": "Continuous integration with Jenkins - Tutorial"
+            },
+            {
+              "description": "Continuous Integration with Jenkins",
```

Comment:
I would add some more description here to distinct it more from the first link provided. Like "It's a video course that teaches A B and C"